### DISCUSSION TITLE
VS Code Plugin

### DISCUSSION THREAD

**Discussion by driesie** [allure-framework/allure2#2044] [TestOps]
Hi,
I am busy having a look at Allure Testops which is looking really impressive and want to start a trial soon to test out all the features.

My only concern is that we use VSCode and the VSCode plugin seems very limited compared to the JetBrains plugin.

Are there plans to enhance the functionality offered in the VSCode plugin?

**Comment by baev** [comment]
> Are there plans to enhance the functionality offered in the VSCode plugin?

Yes

> My only concern is that we use VSCode and the VSCode plugin seems very limited compared to the JetBrains plugin.

Please specify the features you're missing so we can better prioritise the work.

**Reply by driesie** [reply]
I am specifically referring to the features listed in this document where VSCode does not have the equivalent functionality

https://docs.qameta.io/allure-testops/ecosystem/ide-plugins/

These seem like very powerful features that are missing in the VSCode plugin
1. Creation of a new automated test case
2. Import of all the meta-data of a manual test case for its automation in IDE.

I was watching the full overview video on YouTube (https://www.youtube.com/watch?v=oLgSOlf6vPg) and it was incredible what you could do with the Jetbrains plugin, especially with regards to creation of new automated test cases but it seems with the VSCode plugin you can only upload results to Allure Testops.

I am happy that you said there are plans to enhance the VSCode plugin but when will these extra features be available?

It will be a big motivating factor for us to consider this tool since we use VSCode and not JetBrains.

At the very least it would be very helpful if there are some documentation to show how we can get around the missing functionality in the VSCode plugin until the plugin can handle it.
